packages/fungible-token/bridge-fungible-token/src/main.sw

```diff
@@ -74,8 +75,10 @@ impl MessageReceiver for Contract {
                 register_refund(message_data.from, message_data.token, message_data.amount);
             },
             Result::Ok(amount) => {
+                let subId: SubId = ZERO_B256;
+                let tokenId = sha256((contract_id(), subId));
```

Comment:
[nit] there is a helper function for this in the stdlib now  `std::asset_id::construct_default_asset_id`.